tests/test_models.py

```diff
@@ -35,7 +35,10 @@ def test_translations(self):
         user = Users.objects().first()
         lang = Languages.objects().first()
         text = Sentences.objects().first()
-        Translations(author=user, targetlang=lang,
-                     sentence=text, filename="myFile.3gp").save()
-        assert Translations.objects.count() == 1
+        Translations(
+            author=user,
+            targetlang=lang,
+            sentence=text,
+            audiofile=File(name="myFile.3gp", content="test".encode())).save()
```

Comment:
why File?

Reply:
`File` is just a simple json document. To assign a json object to a field with mongoengine, you have to define a class to map that json object.